obspy/signal/src/lanczos_resampling.c

```diff
@@ -0,0 +1,167 @@
+/*--------------------------------------------------------------------
+# Filename: lanczos_resampling.c
+#  Purpose: Lanczos resampling with different kernels
+#           each point.
+#   Author: Lion Krischer
+# Copyright (C) 2015 Lion Krischer and Martin van Driel
+#---------------------------------------------------------------------*/
+
+/* Windows apparently needs this */
+#define _USE_MATH_DEFINES
+
+#include <math.h>
+
+
+enum lanczos_window_type {
+    LANCZOS = 0,
+    HANNING = 1,
+    BLACKMAN = 2,
+};
+
+
+/* Sinc function */
+double sinc(double x) {
+    if (fabs(x) < 1E-10) {
+        return 1.0;
+    }
+    return sin(M_PI * x) / (M_PI * x);
+}
+
+
+/* Standard Lanczos Kernel */
+double lanczos_kernel(double x, int a) {
+    return sinc(x / (double)a);
+}
+
+
+/* von Hann window or raised cosine window*/
+double hanning_kernel(double x, int a) {
+    return 0.5 * (1.0 + cos(x / (double)a * M_PI));
+}
+
+
+/* Blackman window
+ *
+ * Values are taken from http://mathworld.wolfram.com/BlackmanFunction.html
+ */
+double blackman_kernel(double x, int a) {
+    return 21.0 / 50.0 +
+           0.5 * cos(x / (double)a * M_PI) +
```

Comment:
Yea your probably right. It somehow must be the same window because I test it against the numpy implementation and they use the same definition as in the oppenheimer book.